src/app/app-routing.module.ts

```diff
@@ -1,8 +1,12 @@
 import { NgModule } from '@angular/core';
 import { RouterModule, Routes } from '@angular/router';
+import { HomepageComponent } from './pages/homepage/homepage.component';
 import { RssComponent } from './pages/rss/rss.component';
 
-const routes: Routes = [{ path: 'rss', component: RssComponent }];
+const routes: Routes = [
+  { path: 'home', component: HomepageComponent },
```

Comment:
if homepage is the default page please add a redirect path '' to home

Reply:
fixed